deploy/crds/mongodb.com_mongodb_crd.yaml

```diff
@@ -49,6 +49,46 @@ spec:
             members:
               description: Members is the number of members in the replica set
               type: integer
+            security:
+              description: Security configures security features, such as TLS, and
+                authentication settings for a deployment
+              properties:
+                tls:
+                  description: TLS configuration for both client-server and server-server
+                    communication
+                  properties:
+                    caConfigMapRef:
+                      description: CaConfigMap is a reference to a ConfigMap containing
+                        the certificate for the CA which signed the server certificates
+                        The certificate is expected to be available under the key
+                        "ca.crt"
+                      properties:
+                        name:
+                          type: string
+                      required:
+                      - name
+                      type: object
+                    certificateKeySecretRef:
+                      description: CertificateKeySecret is a reference to a Secret
```

Comment:
Maybe mention that this is a kubernetes.io/tls Secret type

Reply:
@rodrigovalin We don't require the secret to be of type `kubernetes.io/tls`, only that the right fields exist. Is it still worth mentioning?

Reply:
it makes sense to mention that the secret is not expected to be "one type or another", that can't be enforced, as much as I understand, but it does not hurt to tell people that that's exactly the format we expect

Reply:
True! Updated the description now to mention this.